src/garage/sampler/__init__.py

```diff
@@ -16,8 +17,19 @@
 from garage.sampler.worker_factory import WorkerFactory
 
 __all__ = [
-    'BatchSampler', 'Sampler', 'ISSampler', 'singleton_pool', 'LocalSampler',
-    'RaySampler', 'SamplerWorker', 'ParallelVecEnvExecutor', 'VecEnvExecutor',
-    'OffPolicyVectorizedSampler', 'OnPolicyVectorizedSampler', 'WorkerFactory',
-    'Worker', 'DefaultWorker'
+    'BatchSampler',
+    'Sampler',
+    'ISSampler',
+    'singleton_pool',
+    'LocalSampler',
+    'RaySampler',
+    'SamplerWorker',
+    'ParallelVecEnvExecutor',
+    'VecEnvExecutor',
+    'OffPolicyVectorizedSampler',
+    'OnPolicyVectorizedSampler',
+    'WorkerFactory',
+    'Worker',
+    'DefaultWorker',
+    'PEARLSampler',
```

Comment:
can this be achieved by adding a PEARLWorker class @krzentner ?

Anson does a similar pattern for RL2: He implements an RL2 worker which lives in `garage.tf.algos.rl2` rather than the central sampler package.

Reply:
It can definitely be achieved with that.
Alternatively, just put z in `agent_infos`. It's there for a reason.